### DISCUSSION TITLE
How can I contribute to Spacy ?

### DISCUSSION THREAD

**Discussion by punitvara** [explosion/spaCy#7156] [Help: Best practices]
Can anyone help me with topic/feature which I can implement and contribute to Spacy ? I have been working as ML engineer since last two years. But have not contributed much on open source software recently. However, I do have extensive experience working on Embedded system related open source projects. Just looking to start contributing on Spacy. Any help would be appreciated .

**Comment by svlandeg** [comment]
Hi @punitvara, we always welcome community contributions!

You can find a bit more information on contributing [here](https://github.com/explosion/spaCy/blob/master/CONTRIBUTING.md). Some issues are specifically labeled with ["help wanted"](https://github.com/explosion/spaCy/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) or ["help wanted (easy)"](https://github.com/explosion/spaCy/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted+%28easy%29%22) - those are often feature requests that we would definitely like to see in the library, but that we haven't had the time to prioritize ourselves yet.

**Reply by punitvara** [reply]
Thank you. I will look into this help wanted issues.
